apps/google-analytics-4/frontend/src/components/main-app/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import tokens from '@contentful/f36-tokens';
+
+const ACCESSIBILITY_LABEL = 'Analytics line chart';
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip);
+
+ChartJS.defaults.font.size = 12;
+ChartJS.defaults.font.family = tokens.fontStackPrimary;
+ChartJS.defaults.font.weight = '500';
+ChartJS.defaults.borderColor = tokens.gray200;
+ChartJS.defaults.datasets.line.borderColor = tokens.colorPrimary;
+
+interface LineChartProps {
+  dataValues: Array<string | number>;
+  xAxisLabels: Array<string>;
+  tooltipMetricLabel: string;
+  accessibilityLabel: string;
+}
+
+const LineChart = ({
+  dataValues,
+  xAxisLabels,
+  tooltipMetricLabel,
+  accessibilityLabel,
+}: LineChartProps) => {
+  const data = {
+    labels: xAxisLabels,
+    datasets: [
+      {
+        data: dataValues,
+      },
+    ],
+  };
+
+  const options = {
+    responsive: true,
+    plugins: {
+      tooltip: {
+        backgroundColor: tokens.gray900,
+        bodyColor: tokens.colorWhite,
+        padding: 8,
+        titleMarginBottom: 4,
+        titleFont: {
+          size: 9,
+        },
+        bodyFont: {
+          size: 8,
+          weight: '700',
+        },
+        displayColors: false,
+        callbacks: {
+          beforeBody: () => tooltipMetricLabel,
+        },
+      },
+    },
+  };
+
+  const props = {
+    'aria-label': accessibilityLabel,
+  };
+
+  return <Line data={data} options={options} fallbackContent={accessibilityLabel} {...props} />;
```

Comment:
not a big fan of spreading props, but believe this is the only way to spread html props down to the `canvas` element, to support accessibility.

Reply:
seems reasonable to me!